src/test/java/org/takes/rq/CapInputStreamTest.java

```diff
@@ -54,4 +56,35 @@ public void putsCapOnStream() throws IOException {
         );
     }
 
+    /**
+     * CapInputStream can close a stream.
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void closesStream() throws IOException {
+        final InputStream stream = Mockito.mock(InputStream.class);
+        final CapInputStream wrapper = new CapInputStream(
+            stream,
+            0L
+        );
+        wrapper.close();
+        Mockito.verify(stream, Mockito.times(1)).close();
+    }
+
+    /**
+     * CapInputStream can skip on a stream.
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void skipsOnStream() throws IOException {
```

Comment:
@original-brownbear I know that in other tests we have `throws IOException`, but what Yegor recommends to do is to use `throws Exception` instead